Wozza365/GameDevelopment
# 3 improvements

Windows control redesigned. 
To tester:
Drag and drop a new windowSwing class from c++ classes. when you aim on lever and press F lever should move and then window should open.

2nd room puzzle variables restart properly when new game starts.

Fixed bug that caused error in FPC BP when player wanted to use key
